### PR TITLE
Critical Updates Contacts, Emails, Connectors 

### PR DESCRIPTION
**Pull request by itsalyse**
Critical edits to articles in the contacts, emails, and connectors categories. 

Changing domain contacts: https://docs.google.com/document/d/17zq-4LO7buSfOckTjXlDZkuVfA7bDRL2PkN51_ssl88/edit?tab=t.0

Managing your contacts: https://docs.google.com/document/d/1wD0d9NVMHfwLZHHtQG2FrwfldLHltB24megbvC5ikko/edit?tab=t.0#heading=h.mzw4ooyxe54b

Heroku connector (minimal edits): https://docs.google.com/document/d/1StkzjJ7IkSb-_BZYGux4UVtM1iOsK6-4LoqAvQ8Hg1M/edit?tab=t.0#heading=h.d0udlkipcrbp

Netlify connector (minimal edits): https://docs.google.com/document/d/1StkzjJ7IkSb-_BZYGux4UVtM1iOsK6-4LoqAvQ8Hg1M/edit?tab=t.0#heading=h.d0udlkipcrbp

Email forwarding: https://docs.google.com/document/d/1A7fwOSFe85AFyjvuvxwHapGyuGTiK0U-x1_UGuRgg6o/edit?tab=t.0

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/changing-domain-contact.md [hunk 1]
```diff
@@ -29,49 +29,36 @@ If you want to assign a new contact to one of your domains, create the new conta
...
+1.  On the card displaying the contact information of your domain, click <label>Change Contact</label>.
 
-    ![Domain registration details](/files/domain-registration-details.png)
-
-1.  On the card displaying the contact information of your domain, click the <label>Change Contact</label> link.
-
-    ![Change domain contact](/files/change-contacts.png)
-
-1.  You will see a list of all your existing contacts. The one currently assigned to the domain is automatically selected. By selecting a contact, you will see the chosen contact information with its extended attributes. Clicking <label>Next</label> lets you confirm the contact information.
+      <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/changing-domain-contact.md:43]: https://drive.google.com/file/d/1yjNu-i4TAVuZuIfzjGX5WQduULDt4w_i/view?usp=sharing

File: content/articles/changing-domain-contact.md [hunk 2]
```diff
@@ -29,49 +29,36 @@ If you want to assign a new contact to one of your domains, create the new conta
...
+    
+1.  You will see a list of all your existing contacts. The one currently assigned to the domain is automatically selected. By selecting a contact, you will see the chosen contact information with its extended attributes. Click <label>Next</label>.
+1.  You will see a confirmation of the new contact information for the domain. You may see a message that the contact change results in the domain being [locked from transfers for 60 days](/articles/icann-60-day-lock-registrant-change/). Click <label>Change contact</label> to confirm the change.
 
-    ![Change contact preview](/files/change-contact-preview.png)
-
-1.  You will see a confirmation of the new contact information for the domain. If you see a message that the contact change results in the domain being [locked from transfers for 60 days](/articles/icann-60-day-lock-registrant-change/), click the <label>Change Contact</label> link to change the contact.
-
-    ![Change contact confirmation](/files/change-contact-confirmation.png)
+    <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/changing-domain-contact.md:48]: https://drive.google.com/file/d/10jZDX3jzbtBm5g-cpg2qVaaokidScP9g/view?usp=sharing

File: content/articles/changing-domain-contact.md [hunk 3]
```diff
@@ -29,49 +29,36 @@ If you want to assign a new contact to one of your domains, create the new conta
...
 
 <div class="section-steps" markdown="1">
 ##### To change the contact information
 
-1.  Log in to DNSimple with your user credentials.
-1.  On the header, click the <label>Contacts</label> tab, locate the contact you want to change, and click <label>Edit</label>.
+    1.  On the header, click the <label>Contacts</label> tab, locate the contact you want to change, and click <label>Edit</label>.
 
-    ![Edit contact data](/files/edit-existing-contact-1.png)
+        <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/changing-domain-contact.md:61]: https://drive.google.com/file/d/1yjNu-i4TAVuZuIfzjGX5WQduULDt4w_i/view?usp=sharing

File: content/articles/netlify-connector.md [hunk 4]
```diff
@@ -24,37 +24,43 @@ categories:
 
 ## Creating a connection
 
-1. Head to the "Connections" tab for the domain you want to connect to Netlify. From there, click the "Add" button next to the Netlify connector.  
-   ![Create a connection to Netlify](/files/netlify-connector-add-connector.png)
+1. Choose the relevant domain from your Domain List.
+2. Click **Connections** the tab on the left side.
+3. Under **Add Connection**, click **Add** in the **Netlify Connector** section.  
+  <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/netlify-connector.md:30]: https://drive.google.com/file/d/1CZojTDB4hl7iQyz51tdmMIBVJMPJugLV/view?usp=sharing

File: content/articles/heroku-connector.md [hunk 5]
```diff
@@ -25,31 +25,38 @@ categories:
...
-   ![Create a connection to Heroku](/files/heroku-connector-add-connector.png)
+1. Choose the relevant domain from your Domain List.
+2. Click the **Connections** tab on the left side.
+3. Under **Add Connection**, click **Add** in the **Heroku Connector** section.
+   
+  <!--- needs screenshot -->
 
-
-2. Authorize your DNSimple account to give the connector permission to access your domains and DNS records.
+4. Authorize your DNSimple account to give the connector permission to access your domains and DNS records.
```
Comment (itsalyse) [inline content/articles/heroku-connector.md:34]: check numbering

File: content/articles/changing-domain-contact.md [hunk 6]
```diff
@@ -14,75 +14,65 @@ categories:
...
+##### Adding and changing contact information
 
-1.  Log in to DNSimple with your user credentials.
 1.  If you have more than one account, select the relevant one.
 1.  On the header, click the <label>Contacts</label> tab, then click <label>New Contact</label>.
 
-    ![Add contact](/files/change-contact-1.png)
+    <!--- needs screenshot -->
 
 1.  Fill in the form with the new contact information, and click <label>Add contact</label>.
```
Comment (itsalyse) [inline content/articles/changing-domain-contact.md:37]: Check numbering

File: content/articles/changing-domain-contact.md [hunk 7]
```diff
@@ -14,75 +14,65 @@ categories:
...
-1.  On the card displaying the contact information of your domain, click the <label>Change Contact</label> link.
-
-    ![Change domain contact](/files/change-contacts.png)
-
-1.  You will see a list of all your existing contacts. The one currently assigned to the domain is automatically selected. By selecting a contact, you will see the chosen contact information with its extended attributes. Clicking <label>Next</label> lets you confirm the contact information.
-
-    ![Change contact preview](/files/change-contact-preview.png)
+      <!--- needs screenshot -->
+    
+1.  You will see a list of all your existing contacts. The one currently assigned to the domain is automatically selected. By selecting a contact, you will see the chosen contact information with its extended attributes. Click <label>Next</label>.
```
Comment (itsalyse) [inline content/articles/changing-domain-contact.md:45]: Check numbering

File: content/articles/changing-domain-contact.md [hunk 8]
```diff
@@ -14,75 +14,65 @@ categories:
...
-1.  Log in to DNSimple with your user credentials.
-1.  On the header, click the <label>Contacts</label> tab, locate the contact you want to change, and click <label>Edit</label>.
+    1.  On the header, click <label>Contacts</label>, locate the contact you want to change, and click <label>Edit</label>.
 
-    ![Edit contact data](/files/edit-existing-contact-1.png)
+        <!--- needs screenshot -->
 
-1.  Update the information you want to change and confirm.
-1.  You will see your changes confirmed. If a contact validation is required, you will receive an email for each associated domain.
+1.  Update the information and click <label>Update contact</label>.
```
Comment (itsalyse) [inline content/articles/changing-domain-contact.md:63]: Check numbering

File: content/articles/netlify-connector.md [hunk 9]
```diff
@@ -24,37 +24,43 @@ categories:
...
-1. Head to the "Connections" tab for the domain you want to connect to Netlify. From there, click the "Add" button next to the Netlify connector.  
-   ![Create a connection to Netlify](/files/netlify-connector-add-connector.png)
+1. Choose the relevant domain from your Domain List.
+2. Click the **Connections** tab on the left side.
+3. Under **Add Connection**, click **Add** in the **Netlify Connector** section.  
+  <!--- needs screenshot -->
 
-
-2. Authorize your DNSimple account to give the connector permission to access your domains and DNS records.
+4. Authorize your DNSimple account to give the connector permission to access your domains and DNS records.
```
Comment (itsalyse) [inline content/articles/netlify-connector.md:32]: Check numbering

File: content/articles/email-forwarding.md [hunk 10]
```diff
@@ -20,31 +20,32 @@ The number of forwarded messages and email forwards allowed per month is based o
...
 
-To enable email forwarding, visit a domain's manage page and click on the **Email forwarding** menu on the sidebar.
+To enable email forwarding, 
 
-![Email forwarding](/files/email-forwarding.png)
+1. Choose the relevant domain from your Domain List.
+2. Click the **Email Forwarding** tab menu on left side. 
 
-Create an email forward to enable the service.
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/email-forwarding.md:28]: https://drive.google.com/file/d/1BAC7FbzAA7tGSIz197qzVP34YnVDzEbt/view?usp=sharing

File: content/articles/email-forwarding.md [hunk 11]
```diff
@@ -20,31 +20,30 @@ The number of forwarded messages and email forwards allowed per month is based o
...
+1. Choose the relevant domain from your Domain List.
+2. Click the **Email Forwarding** tab on the left side. 
 
-![Email forwarding](/files/email-forwarding.png)
+<!--- needs screenshot -->
 
-Create an email forward to enable the service.
-
-When you enable email forwarding, we will automatically add the various DNS records needed to support forwarding.
+3. Create an email forward to enable the service.
```
Comment (itsalyse) [inline content/articles/email-forwarding.md:28]: check numbering

**Comment by itsalyse** [comment]
@dperrymorrow this is ready for screenshots!

File: content/articles/changing-domain-contact.md [hunk 12]
```diff
@@ -14,86 +14,79 @@ categories:
...
+For domains registered with DNSimple, [Domain Managers](https://support.dnsimple.com/articles/domain-access-control/#domain-manager) and anyone with [Full Access](https://support.dnsimple.com/articles/domain-access-control/#full-access) can change the contact associated with a domain at any time.
 
-<note>
-#### Domain Not Registered with DNSimple
+<info>
+#### Domains not registered with DNSimple
 
-The following instructions apply only to a domain registered with DNSimple. If you are hosting the domain with us but the domain is registered elsewhere, you will have to transfer it or update the contact information at your current registrar.
-</note>
+These instructions apply only to domains registered with DNSimple. If you are hosting the domain with us, and the is domain registered elsewhere, you will have to update the contact information at your current registrar, or transfer the domain to DNSimple.
```
Comment (stephanoodle) [inline content/articles/changing-domain-contact.md:22]: > If you are hosting the domain with us, and the is domain registered elsewhere

need to move the "is" after "domain" for this to make sense.

**Review by stephanoodle — APPROVED** [review]
Once the wording I found is fixed, this is good to go for me!

**Review by stephanoodle — APPROVED** [review]

File: content/articles/email-forwarding.md (hunk 10 above)
Comment (Pemacaa) [reply content/articles/email-forwarding.md:26]: The screenshot shows the "Connections" section. Am I looking wrong?

File: content/articles/email-forwarding.md [hunk 13]
```diff
@@ -16,35 +16,34 @@ categories:
...
 
 ## Creating an email forward
 
 <div class="section-steps" markdown="1">
 ##### Adding an email forward from the dashboard
 
-1.  Log in to DNSimple with your user credentials.
 1.  If you have more than one account, select the relevant one.
 1.  Click <label>Add</label> from the Dashboard, and choose <label>Email forward</label> from the provided options.
   ![Adding an email forward](/files/add-button-email-forward.png)
```
Comment (Pemacaa) [inline content/articles/email-forwarding.md:39]: The Screenshot is outdated

File: content/articles/email-forwarding.md [hunk 14]
```diff
@@ -16,35 +16,34 @@ categories:
...
 </div>
 
-To create an email forward you need to specify the **From** and **To** fields.
+To create an email forward, specify the **From** and **To** fields.
 
 ![Email forwarding creation](/files/email-forwarding-setup.png)
```
Comment (Pemacaa) [inline content/articles/email-forwarding.md:44]: This screenshot is also outdated

Comment (itsalyse) [reply content/articles/email-forwarding.md:44]: Also looks exactly the same as it does in the sandbox.

File: content/articles/email-forwarding.md (hunk 13 above)
Comment (itsalyse) [reply content/articles/email-forwarding.md:39]: Shot from just now 
Screenshot 2024-11-21 at 10 16 40 AM

File: content/articles/email-forwarding.md (hunk 10 above)
Comment (itsalyse) [reply content/articles/email-forwarding.md:26]: yep that's the wrong one will replace

File: content/articles/email-forwarding.md (hunk 14 above)
Comment (itsalyse) [reply content/articles/email-forwarding.md:44]: ah there's an extra caveat - if that's important to have here we can change this.

File: content/articles/email-forwarding.md (hunk 13 above)
Comment (Pemacaa) [reply content/articles/email-forwarding.md:39]: Oh, I wasn't looking at the correct file then

File: content/articles/email-forwarding.md (hunk 14 above)
Comment (itsalyse) [inline content/articles/email-forwarding.md:44]: https://drive.google.com/file/d/1KS-SG2RwgjFr-vTYWFCoRjgM5s8C31tY/view?usp=sharing

File: content/articles/email-forwarding.md [hunk 15]
```diff
@@ -16,35 +16,34 @@ categories:
...
 1.  If you have more than one account, select the relevant one.
 1.  Click <label>Add</label> from the Dashboard, and choose <label>Email forward</label> from the provided options.
   ![Adding an email forward](/files/add-button-email-forward.png)
 </div>
 
-To create an email forward you need to specify the **From** and **To** fields.
+To create an email forward, specify the **From** and **To** fields.
 
-![Email forwarding creation](/files/email-forwarding-setup.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/email-forwarding.md:44]: https://drive.google.com/file/d/1KS-SG2RwgjFr-vTYWFCoRjgM5s8C31tY/view?usp=sharing

**Comment by dperrymorrow** [comment]
Ill work on this PR today

File: content/articles/changing-domain-contact.md [hunk 16]
```diff
@@ -14,86 +14,75 @@ categories:
...
+You can update the contact details to change existing contact information. DNSimple will immediately update the registry data and the WHOIS record with the new information.
 <div class="section-steps" markdown="1">
 ##### To change the contact information
 
-1.  Log in to DNSimple with your user credentials.
-1.  On the header, click the <label>Contacts</label> tab, locate the contact you want to change, and click <label>Edit</label>.
-
-    ![Edit contact data](/files/edit-existing-contact-1.png)
+1.  On the header, click <label>Contacts</label>, locate the contact you want to change, and click <label>Edit</label>.
+    ![screenshot of editing domain contact](/files/registration-edit-contact.png)
```
Comment (itsalyse) [inline content/articles/changing-domain-contact.md:56]: @dperrymorrow this is the wrong screenshot, should be https://drive.google.com/file/d/1NVmoae4G9bXBC4kVQtg8uZqxqq08EXdr/view?usp=sharing

File: content/articles/changing-domain-contact.md [hunk 17]
```diff
@@ -14,86 +14,75 @@ categories:
...
+You can update the contact details to change existing contact information. DNSimple will immediately update the registry data and the WHOIS record with the new information.
 <div class="section-steps" markdown="1">
 ##### To change the contact information
 
-1.  Log in to DNSimple with your user credentials.
-1.  On the header, click the <label>Contacts</label> tab, locate the contact you want to change, and click <label>Edit</label>.
-
-    ![Edit contact data](/files/edit-existing-contact-1.png)
+1.  On the header, click <label>Contacts</label>, locate the contact you want to change, and click <label>Edit</label>.
+   <!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/changing-domain-contact.md:56]: @dperrymorrow the wrong screenshot was added, I've reverted to the tag — it should be 
https://drive.google.com/file/d/1NVmoae4G9bXBC4kVQtg8uZqxqq08EXdr/view?usp=sharing

File: content/articles/email-forwarding.md [hunk 18]
```diff
@@ -16,35 +16,34 @@ categories:
...
+The number of forwarded messages and email forwards allowed per month is based on your plan. [See the differences between plans](https://dnsimple.com/pricing).
 
 ## Enabling email forwarding
 
-To enable email forwarding, visit a domain's manage page and click on the **Email forwarding** menu on the sidebar.
+1. Choose the relevant domain from your Domain List.
+2. Click the **Email Forwarding** tab on the left side. 
 
-![Email forwarding](/files/email-forwarding.png)
+<!--- needs screenshot -->
```
[the remark this answers is not in the document]
Comment (itsalyse) [reply content/articles/email-forwarding.md:26]: @dperrymorrow this one wasn't added

File: content/articles/email-forwarding.md (hunk 15 above)
Comment (itsalyse) [reply content/articles/email-forwarding.md:44]: @dperrymorrow this one wasn't added

File: content/articles/heroku-connector.md [hunk 19]
```diff
@@ -25,31 +25,35 @@ categories:
 
 ## Creating a connection
 
-1. Head to the "Connections" tab for the domain you want to connect to Heroku. From there, click the "Add" button next to the Heroku connector.
-   ![Create a connection to Heroku](/files/heroku-connector-add-connector.png)
+1. Choose the relevant domain from your Domain List.
+2. Click the **Connections** tab on the left side.
+3. Under **Add Connection**, click **Add** in the **Heroku Connector** section.
+    ![screenshot of adding a connector](/files/add-connector.png)
```
Comment (itsalyse) [inline content/articles/heroku-connector.md:31]: @dperrymorrow I accidentally used the netlify one here, should be @dperrymorrow this is the wrong screenshot, should be https://drive.google.com/file/d/1NVmoae4G9bXBC4kVQtg8uZqxqq08EXdr/view?usp=sharing

**Pull request merged**